src/v/json/_include_first.h

```diff
@@ -11,7 +11,12 @@
 
 #pragma once
 
-#include "base/vassert.h"
+#include <seastar/util/backtrace.hh>
 
 #define RAPIDJSON_HAS_STDSTRING 1
-#define RAPIDJSON_ASSERT(x) vassert(x, "Rapidjson ")
+#define RAPIDJSON_ASSERT(x)                                                    \
```

Comment:
why not use vassert ?, specially if you are importing seastar now

Reply:
to give an example: in tx_range_manifest, deserialization is done without validating the schema, an input like `{}` would trigger the assert. 
with this change, the process does not stop. at least for unit testing, it's a better experienceù